### PR TITLE
Add TWS v6 to servers_v6.json

### PR DESCRIPTION
**Pull request by Neba-MD**
Administration is ready.

### CODE REVIEW

File: servers_v6.json [hunk 1]
```diff
@@ -31,6 +31,10 @@
     "name": "Ranked",
     "address": ["mindustryranked.ddns.net:16567"]
   },
+  {
+    "name": "TWS",
+    "address": ["twsmindustry.24x7.hk:6701", "twsmindustry.24x7.hk:6702]
```
Comment (Anuken) [inline servers_v6.json:36]:
```suggestion
    "address": ["twsmindustry.24x7.hk:6701", "twsmindustry.24x7.hk:6702"]
```

Missing quote. This isn't valid JSON.

**Comment by DeltaNedas** [comment]
1. use old plugin
2. best

**Comment by Neba-MD** [comment]
Sorry, should be there now.

**Comment by Neba-MD** [comment]
The servers are also updated to build 121 if it is part of your consideration.

**Pull request merged**